### PR TITLE
Bug 1161201 accept input of negative adjacency inclusion

### PR DESCRIPTION
**Pull request by oyiptong**
tracked at: https://bugzilla.mozilla.org/show_bug.cgi?id=1161201

### CODE REVIEW

**Comment by oyiptong** [comment]
r? @ncloudioj 

This PR is rebased on top of #62

**Comment by ncloudioj** [comment]
Shouldn't we use two separate PRs for two features?

File: tests/base.py [hunk 1]
```diff
@@ -32,10 +32,10 @@ def tile_values(fd):
 
         def adgroup_values(fd):
             for line in fd:
-                row = [el.decode('utf-8') for el in line.split(',')]
+                locale, check_inadjacency_str = [el.decode('utf-8') for el in line.split(',')]
                 yield dict(zip(
-                    ('locale',),
-                    row))
+                    ('locale', 'check_inadjacency'),
+                    (locale, True if check_inadjacency_str == 'true' else False)))
```
Comment (tspurway) [inline tests/base.py]: nit:  
True if check_inadjacency_str == 'true' else False
is the same as
check_inadjacency_str == 'true'

Comment (oyiptong) [inline tests/base.py]: Where is my mind? :-)

**Comment by tspurway** [comment]
r+ @oyiptong

**Pull request closed without merging**